### ISSUE TITLE
*Bug* Can not add new entries in existing outbound route

### ISSUE THREAD

**Issue by andycol** [fusionpbx/fusionpbx#1757]
When you try add a new line in an existing outbound route it says saved but shows no record?
example

i add set action pin_number=database 61 and click save
the option does not appear

this is a major issue as now we can not use toll fraud or pin_numbers

**Comment by moteus** [comment]
Seems new `orm` class does not set correctly `dialplan_uuid` field when do insert into `v_dialplan_details`
To check just execute SQL after add new action.
```SQL
select * from v_dialplan_details where dialplan_uuid is null
```

**Comment by markjcrane** [comment]
This bug should be fixed dd2e18fbdd6107b21047cc31d7a5bf4be8352845. Please update and test.

**Comment by andycol** [comment]
updated and all is now working as expected.

Thanks again Mark
